projects/plugins/boost/tests/e2e/lib/pages/wp-admin/JetpackBoostPage.js

```diff
@@ -47,6 +47,11 @@ export default class JetpackBoostPage extends WpPage {
 		this.page.click( `#jb-feature-toggle-${ moduleName }` );
 	}
 
+	async toggleModuleAndWaitForResponse( moduleName ) {
```

Comment:
Do we ever call `toggleModule` without waiting for a response?

I'm inclined to suggest that we just update `toggleModule` to also wait for the API response, and not have a big, unwieldy function name with an `and` in it.